cartoframes/context.py

```diff
@@ -1553,30 +1549,40 @@ def data(self, table_name, metadata, persist_as=None, how='the_geom'):
                              'combine resulting DataFrames using '
                              '`pandas.concat`')
 
-        tablecols = self.sql_client.send(
-            'SELECT * FROM {table_name} LIMIT 0'.format(table_name=table_name),
-            **DEFAULT_SQL_ARGS
-        )['fields'].keys()
+        # get column names except the_geom_webmercator
+        dataset = Dataset(self, table_name)
+        table_columns = dataset.get_table_column_names(exclude=['the_geom_webmercator'])
```

Comment:
FYI, the `DEFAULT_SQL_ARGS` are the way to get the cartoframes calls to show up in logs so we can monitor performance, etc. They're fit in so that each method get registered once despite having multiple calls.

Reply:
mmm is this still true? I see DEFAULT_SQL_ARGS just having a `do_post` attribute.

Reply:
Good question. Are POST request args searchable in kibana?

Reply:
@oleurud ^^

Reply:
The POST arguments are not saved by default. The only things we can check in Kibana about a POST request to SQL API are the queries

Reply:
Two options:
- Use the user agent header, which is designed to do exactly this (and it's how we do it in carto-python)
- Add a SQL comment to the queries (seems hacky)

I'd do the user agent thing, which should not be hard to do and it makes it searchable everywhere: Kibana, Rollbar, etc.

Reply:
Just in case it will be solved with a user agent header, I had created this issue the handle it https://github.com/CartoDB/cartoframes/issues/601

Reply:
I added the user-agent thing [here](https://github.com/CartoDB/cartoframes/pull/592/commits/9c73e212e4aeec991ef830fc02d005d7dba0177d)

Reply:
Also (https://github.com/CartoDB/carto-python/pull/111). 

Nice, well solved :+1:

Reply:
👏